www/api.html

```diff
@@ -1,101 +1,4 @@
-
-  <p>
-    <code>paper.get()</code> takes a name of something to retrieve, and returns a promise, so you can use it with <code>await</code>. (It also takes a callback if you prefer that.)
-  </p>
-
-  <p>
-    <code>const canvas = await paper.get('canvas');</code> will give you a <a href="https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API"><code>Canvas</code></a> object to render to. You can ask for a canvas with a certain size, e.g. <code>const canvas = await paper.get('canvas', { width: 500 });</code> You can also get a canvas that's projected onto another program, like this: <code>const canvas = await paper.get('canvas', { number: 1234 });</code>
-  </p>
-
-  <p>
-    <code>const supporterCanvas = await paper.get('supporterCanvas');</code> will give you a <a href="https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API"><code>Canvas</code></a> object that spans the entire projection screen. You can get multiple of them, using: <code>const supporterCanvas = await paper.get('supporterCanvas', { id: 'blah' });</code>
-  </p>
-
-  <p>
-    <code>const papers = await paper.get('papers');</code> will give you an object that looks something like this:<br>
-    <code>
-      {<br>
-      &nbsp;&nbsp;"512": {<br>
-      &nbsp;&nbsp;&nbsp;&nbsp;"data": {},<br>
-      &nbsp;&nbsp;&nbsp;&nbsp;"points": { "center": { x: 100, y: 200 }, "topLeft": ..., ... }<br>
-      &nbsp;&nbsp;}<br>
-      }
-    </code>
-  </p>
-
-  <p>
-    <code>const number = await paper.get('number');</code> gives you your own paper number, so you can look up your <code>"data"</code> and <code>"points"</code> in the list of papers above.
-  </p>
-
-  <p>
-    <code>await paper.set('data', { someKey: 'someValue' });</code> will add stuff to the <code>"data"</code> object above, which will be visible to all programs. It will even persist if a program stops running.
-  </p>
-
-  <p>
-    <code>await paper.set('iframe', { src: 'http://www.example.com' });</code> will cover your page in an iframe with source at that url.
-  </p>
-
-  <h3>paper.whenPointsAt()</h3>
-
-  <p>
-    This is a helper function for creating papers that point at other papers. You must supply a callback which will be called when the paper points at a different paper (or none at all). <code>paper.whenPointsAt({ callback: function (pointAtData) => { /* do something */ } });</code>
-  </p>
-
-  <p>
-    Other options are:
-    <ul>
-      <li><code>direction</code>: "up" (default), "down", "left", "right";</li>
-      <li><code>whiskerLength</code>: as fraction of the side (default 0.7);</li>
-      <li><code>requiredData</code>: array of data fields that must be present in the other paper;</li>
-      <li><code>paperNumber</code>: paper number to do this for (default is own paper number).</li>
-    </ul>
-  </p>
-</body>
+<script>
+  // Redirect to new API documentation page:
+  window.location.replace("https://github.com/janpaul123/paperprograms/blob/master/docs/api.md");
+</script>
```

Comment:
Maybe just delete this page entirely and link directly to `https://github.com/janpaul123/paperprograms/blob/master/docs/api.md` from the editor page?

Reply:
Sure, though I doubt that many people will have this bookmarked since it's right there on the editor page. Let's at least then also update the link though, no need to link to an old page.

Reply:
yeah, maybe I'm on the only one who bookmarked it.  just updated the editor link 👍

Reply:
Thanks!